nautobot/docs/user-guide/administration/migration/migrating-from-postgresql.md

```diff
@@ -15,10 +15,6 @@ nautobot-server dumpdata \
     > nautobot_dump.json
 ```
 
-+/- 1.5.23
-    - We do not recommend at this time using `--natural-primary` as this can result in inconsistent or incorrect data for data models that use GenericForeignKeys, such as `Cable`, `Note`, `ObjectChange`, and `Tag`.
-    - We also do not recommend at this time using `--natural-foreign` as it can potentially result in errors if any data models incorrectly implement their `natural_key()` and/or `get_by_natural_key()` API methods.
-    - `contenttypes` must not be excluded from the dump (it could be excluded previously due to the use of `--natural-foreign`).
```

Comment:
Change to a `!!! warning` instead of removing?

Reply:
Makes sense, adding in as a new warning, though I could also collapse it in to the warning block below if that is preferred.

Reply:
It's a distinct issue compared to the below warning, so I'd probably vote to keep it separate, though I recognize that is a bit awkward.